app/portainer/helpers/webhookHelper.js

```diff
@@ -4,9 +4,9 @@ angular.module('portainer.app')
   var helper = {};
 
   helper.returnWebhookUrl = function(token) {
-    var displayPort = $location.protocol().toLowerCase() == "http" && $location.port() == 80 || $location.protocol().toLowerCase() == "https" && $location.port() == 443 ? "" : ":" + $location.port();
-    return $location.protocol() + "://" +  $location.host()
-    + displayPort + "/" + API_ENDPOINT_WEBHOOKS + "/" + token;
+    var displayPort = $location.protocol().toLowerCase() === 'http' && $location.port() === 80 || $location.protocol().toLowerCase() === 'https' && $location.port() === 443 ? '': ':' + $location.port();
```

Comment:
Should `? '': ':' +` be `? '' : ':' +` instead? Note the space before the first `:`.